docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -1,39 +1,180 @@
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.OnlyIfNotHandled.Ruleset]]
+```
+
+  | onlyIfNotHandled | Result                                                                                                         |
+  | ---------------- | -------------------------------------------------------------------------------------------------------------- |
+  | `true`           | ![Example when selecting all instances](./media/contentinstancesofspecificclasses-with-onlyifnothandled-1.png) |
+  | `false`          | ![Example when filtering instances](./media/contentinstancesofspecificclasses-with-onlyifnothandled-2.png)     |
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Defines the order in which specifications are handled - higher priority means the specifications is handled first. If priorities are equal, the specifications are handled in the order they're defined.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.Priority.Ruleset]]
+```
+
+![Example of using priority attribute](./media/contentinstancesofspecificclasses-with-priority.png)
+
+### Attribute: `relatedProperties`
+
+Specifications of [related properties](./RelatedPropertiesSpecification.md) which are included in the generated content.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.RelatedProperties.Ruleset]]
+```
+
+  |         | Result                                                                                                                    |
+  | ------- | ------------------------------------------------------------------------------------------------------------------------- |
+  | without | ![Example when doing normal property select](./media/contentinstancesofspecificclasses-with-relatedproperties-1.png)      |
+  | with    | ![Example when selecting with related properties](./media/contentinstancesofspecificclasses-with-relatedproperties-2.png) |
+
+### Attribute: `calculatedProperties`
+
+Specifications of [calculated properties](./CalculatedPropertiesSpecification.md) whose values are generated using provided [ECExpressions](../Advanced/ECExpressions.md#ecinstance).
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.CalculatedProperties.Ruleset]]
+```
+
+![Example of using calculatedProperties attribute](./media/contentinstancesofspecificclasses-with-calculatedproperties.png)
+
+### Attribute: `propertyCategories`
+
+Specifications of [custom categories](PropertyCategorySpecification.md).
+
+Simply defining the categories does nothing - they have to be referenced through [`PropertySpecification.categoryId`](./PropertySpecification.md) specified in [`propertyOverrides`](#attribute-propertyoverrides) list.
+
+```ts
+[[include:ContentInstancesOfSpecificClasses.PropertyCategories.Ruleset]]
+```
+
+![Example of using propertyCategories attribute](./media/contentinstancesofspecificclasses-with-propertycategories.png)
```

Comment:
Problem with the screenshot is that it shows property grid content for multiple elements and the values get merged. I think we need to produce a case where content is displayed for just 1 element.

Reply:
what if I take example where all three instances share the same values? Filtering to take 1 element with a specification that is designed to take a whole class of elements also feels like a bad example (I would even argue that the whole attribute feels misplaced on specifications which return many instances).